### DISCUSSION TITLE
Build Stockfish for x86-64 Android

### DISCUSSION THREAD

**Discussion by topse** [official-stockfish/Stockfish#4873] [General]
Hi,
For development purposes it would be really handy to have a Stockfish for x86-64 Android which then could be run in the Debugger. I tried to make one with
```make -j build ARCH=x86-64 COMP=ndk```
But the build-process fails immediatly.

Is there a way?

Thanks and Best Regards,
Tobias

**Comment by Craftyawesome** [comment]
I'm not sure it would be useful to debug the android build, but it seems like you just have to add the arch to the makefile in the right spot
https://github.com/Craftyawesome/Stockfish/commit/bd997a22b57e04ba5a3293db014d037b1af9b24a
It builds fine but I didn't test the binary.

**Reply by topse** [reply]
Sorry -- I do not want to debug Stockfish but to develop a GUI. Therefor it's nice to work in the Emulator and have the engine available.

I downloaded your Repository. Could you give me the make-command? The one on top does not work :-/

**Reply by Craftyawesome** [reply]
Strange, that is what compiled on my end. Did you get an arm compile working and are you sure you are on the correct branch of my repo?

**Reply by topse** [reply]
Sorry... I must have done something wrong... Executable works fine in Android Emulator on x86 Windows Machine!! Many Thanks and Best Regards!!
